server/src/main/java/module-info.java

```diff
@@ -461,7 +461,8 @@
             org.elasticsearch.index.codec.Elasticsearch814Codec,
             org.elasticsearch.index.codec.Elasticsearch816Codec,
             org.elasticsearch.index.codec.Elasticsearch900Codec,
-            org.elasticsearch.index.codec.Elasticsearch900Lucene101Codec;
+            org.elasticsearch.index.codec.Elasticsearch900Lucene101Codec,
+            org.elasticsearch.index.codec.Elasticsearch902Lucene103Codec;
```

Comment:
that makes me wonder why I used 900 instead of 90 back when I added it in the first place. 9.2.0 could have been 92 as well and perhaps 902 makes it look like a bugfix release which it is not.

Reply:
Oh good point. So this should be `Elasticsearch92Lucene103Codec`, then right ? I'll update it.

Reply:
I think it would be better yea. pardon the inconsistency with 900 that we will have to live with.